docs/topics/coroutine-context-and-dispatchers.md

```diff
@@ -480,11 +499,14 @@ I'm working in thread DefaultDispatcher-worker-1 @test#2
 
 ## Coroutine scope
 
-Let us put our knowledge about contexts, children and jobs together. Assume that our application has
-an object with a lifecycle, but that object is not a coroutine. For example, we are writing an Android application
-and launch various coroutines in the context of an Android activity to perform asynchronous operations to fetch 
-and update data, do animations, etc. All of these coroutines must be cancelled when the activity is destroyed
-to avoid memory leaks. We, of course, can manipulate contexts and jobs manually to tie the lifecycles of the activity 
+Let us put our knowledge about contexts, children, and jobs together.
+Assume that our application has an object with a lifecycle, but that object is not a coroutine.
+For example,
+we are writing an Android application, 
+and we launched various coroutines in the context of an Android activity to perform asynchronous operations to fetch 
```

Comment:
The past tense doesn't fit here, as this is something that happens constantly throughout the lifecycle of the application. We can say "we launch" (meaning "we constantly launch"), or "we have to launch", or "various coroutines need to run", or something like that.